incubator/nodejs-express/README.md

```diff
@@ -1,79 +1,65 @@
 # Node.js Express Stack
 
-This provides a prototype platform definition for Node.js Express based projects. This prototype uses Node.js v10 and Express v4.16.0
+The Node.js Express stack extends the [Node.js stack](../nodejs/README.md) and provides a consistent way of developing web applications using [Express](https://expressjs.com). Express is a fast, minimal and flexible Node.js web application framework that provides a robust set of features for web and mobile applications.
 
-## Stacks
+This stack is based on `Node.js v10` and `Express v4.16.0` and includes enables health checking and application metrics out of the box. The stack also provides a set of tests written using [Mocha](https://mochajs.org) test framework while the application metadata is provided via a `package.json` file.
 
-A stack is the foundation of your application.
+## Health checking
 
-The following capabilities are available form the Node.js Express Stack:
-
-### Health checking
-
-Health-checking enables the cloud platform to determine `readiness` (is your application ready to receive requests?) and `liveness` (is your application live or does it need to be restarted?) of your application.
+Health-checking enables the cloud platform to determine `readiness` (is your application ready to receive requests?) and `liveness` (is your application alive or does it need to be restarted?) of your application.
 
 The [@cloudnative/cloud-health](https://github.com/CloudNativeJS/cloud-health) module is a core library that provides health checking and gives you more control over application lifecycle management (startup, running, shutdown). The [@cloudnative/health-connect](https://github.com/CloudNativeJS/cloud-health-connect) module exposes the health-check information on readiness/liveness endpoints to help the cloud platforms manage your application.
 
-You can override or enhance the following endpoints by configuring your own health checks in `app.js`. If you do not specify a health endpoint thes endpoints will be available for you to use:
+ These endpoints will be available for you to use:
+
+- Readiness endpoint: http://localhost:3000/ready
+- Liveness endpoint: http://localhost:3000/live
+- Health check endpoint: http://localhost:3000/health
 
-- Readiness Check: http://localhost:3000/ready
-- Liveness Check: http://localhost:3000/live
-- Health Check: http://localhost:3000/health
+You can override or enhance the following endpoints by configuring your own health checks in your application.
 
-### Application metrics
+## Application metrics
 
 Enable powerful monitoring for your distributed application and configure rule-based alerting using Prometheus open source project. This is vital for diagnosing problems and ensuring the reliability of your application.
```

Comment:
"using the Prometheus open source project" or "using Prometheus".

Reply:
addressed